cmd/rac/output.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+func info() {
+	println(`
+### CSVs ###
+
+The first row:
+- "CODE": Means the following column says what version of the code produced
+  the output.
+- "v0.2.2": The code version
+- "RAMES": Means the following column says what RAMSES specification was used.
+- "SPU045-S2:6F": The RAMSES version
+- "INNOSAT": Means the following column says what INNOSAT specification was
+  used.
+- "IS-OSE-ICD-0005:1": The INNOSAT version
+- "AEZ": Means the following column says what AEZ specification was used.
+- "AEZICD002:H": The AEZ version
+
+The header row starts with a couple of columns common to all output and then
+follows columns specific to each file
+
+
+- File
+  The full path to the rac-file on the computer that produced the csv
+- ProcessingDate
+  The local time when the file was processed
+- RamsesTime (Ramses Header)
+  The time when the ramses file was created (UTC)
+
+- QualityIndicator (Ramses TM Header)
+  Indicates whether the transported data is complete or partial
+  (0 = Complete, 1 = partial).
+- LossFlag (Ramses TM Header)
+  Used to indicate that a sequence discontinuity has been detected
+- VCFrameCounter (Ramses TM Header)
+  Counter of the transfer frame the payload packet arrived in.
+
+- SPSequenceCount (Innosat Source Header)

```

Comment:
```suggestion
- SPSequenceCount (Innosat Source Header)
  A counter that increases with each packet, may never short cycle and should wrap around to zeron after 2^14-1

```

Reply:
@joakimmoller is that the correct counter description?

Reply:
`s/zeron/zero/`

Reply:
Correct description of SPSSequenceCount. You could also mention that the VCFrameCounter wraps at (2^16)-1 i.e. (modulo 65536)